src/components/Garden/Feed/AbstainCard.tsx

```diff
@@ -45,17 +45,53 @@ function AbstainCard({ proposal }: AbstainCardProps) {
         `}
       `}
     >
-      <ProposalInfo
-        proposal={proposal}
-        onSelectProposal={handleSelectProposal}
-        loading={false}
-      />
-      <ProposalFooter
+      <AbstainCardHeader
         proposal={proposal}
-        onSelectProposal={handleSelectProposal}
+        handleSelectProposal={handleSelectProposal}
       />
+      <div onClick={handleSelectProposal} style={{ marginTop: '16px' }}>
+        <ProposalSupport proposal={proposal} isAbstainProposal={true} />
```

Comment:
I think too its possible use just `isAbstainProposal` instead `isAbstainProposal={true}` but i dont mind. Commenting for those who didn't know.

Reply:
Yes I know it can be used like that. But I like that way for those with less experience ✌️